src/main/java/frc/robot/constants.java

```diff
@@ -0,0 +1,131 @@
+package frc.robot;
+
+import com.revrobotics.CANSparkBase.IdleMode;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.kinematics.SwerveDriveKinematics;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.math.util.Units;
+
+public class constants {
+    public static final class DriveConstants {
+        // Driving Parameters - Note that these are not the maximum capable speeds of
+        // the robot, rather the allowed maximum speeds
+        public static final double kMaxSpeedMetersPerSecond = 4.8;
+        public static final double kMaxAngularSpeed = 2 * Math.PI; // radians per second
+
+        public static final double kDirectionSlewRate = 1.2; // radians per second
+        public static final double kMagnitudeSlewRate = 1.8; // percent per second (1 = 100%)
+        public static final double kRotationalSlewRate = 2.0; // percent per second (1 = 100%)
+
+        // Chassis configuration
+        public static final double kTrackWidth = Units.inchesToMeters(25.0 - 3.5);
+        // Distance between centers of right and left wheels on robot
+        public static final double kWheelBase = Units.inchesToMeters(34.0 - 3.5);
```

Comment:
As of two seconds ago we decided it is going to be 28" x 22", don't you know how to time travel?